tartufo/commands/pre_commit.py

```diff
@@ -1,23 +1,25 @@
 import pathlib
-from typing import List, Tuple
+from typing import Optional, Tuple
 
 import click
 
 from tartufo import types, util
-from tartufo.scanner import GitPreCommitScanner, Issue
+from tartufo.scanner import GitPreCommitScanner
 
 
 @click.command("pre-commit")
 @click.pass_obj
 @click.pass_context
-def main(ctx: click.Context, options: types.GlobalOptions) -> Tuple[str, List[Issue]]:
+def main(
+    ctx: click.Context, options: types.GlobalOptions
+) -> Tuple[str, Optional[GitPreCommitScanner]]:
```

Comment:
I wonder... is this actually `Optional`? It seems to me like we're always returning a scanner. And really we should always be. Or was this the result of `mypy` complaining about the `scanner = None`?

Reply:
You're right, it's the second one, `mypy` complaining about `scanner = None`

Reply:
Oof. We'll have to figure out how to fix that (in another PR), because this return type is inaccurate, which could to potential side-effects elsewhere.